cmd/zed/dev/zngdump/command.go

```diff
@@ -0,0 +1,180 @@
+package zngdump
+
+import (
+	"bufio"
+	"encoding/binary"
+	"errors"
+	"flag"
+	"fmt"
+	"io"
+	"os"
+
+	"github.com/brimdata/zed/cmd/zed/dev"
+	"github.com/brimdata/zed/cmd/zed/root"
+	"github.com/brimdata/zed/pkg/charm"
+	"github.com/brimdata/zed/zcode"
+)
+
+var Cmd = &charm.Spec{
+	Name:  "zngdump",
+	Usage: "zngdump file",
+	Short: "prints framing information of ZNG",
+	Long: `
+This command is used for test and debug of the ZNG format.`,
+	New: New,
+}
+
+func init() {
+	dev.Cmd.Add(Cmd)
+}
+
+type Command struct {
+	*root.Command
+	from int
+	to   int
+}
+
+func New(parent charm.Command, f *flag.FlagSet) (charm.Command, error) {
+	c := &Command{Command: parent.(*root.Command)}
+	f.IntVar(&c.from, "from", 0, "seek range from")
+	f.IntVar(&c.to, "to", 0, "seek range to")
+	return c, nil
+}
+
+func (c *Command) Run(args []string) error {
+	if len(args) != 1 {
+		return errors.New("zngdump: one file name argument required")
+	}
+	f := os.Stdin
+	fname := args[0]
+	if fname != "-" {
+		var err error
+		f, err = os.Open(fname)
+		if err != nil {
+			return err
+		}
+	}
+	if c.from != 0 || c.to != 0 {
+		chunk := io.NewSectionReader(f, int64(c.from), int64(c.to)-int64(c.from))
+		_, err := io.Copy(os.Stdout, chunk)
+		return err
+	}
```

Comment:
This is weird and doesn't feel like it belongs here. I expected -from and -to to determine the range fed to the reader below, so maybe do that instead?

Reply:
This is an early start to more useful tooling and I'd rather make it better later.  I needed to use this and found out dd also does the trick so I will take it out.

Reply:
Actually, let's just leave it and fix it later.